.github/workflows/test.yml

```diff
@@ -73,9 +73,35 @@ jobs:
 
   windows:
     name: "Windows tests"
-    runs-on: windows-2022
+    runs-on: windows-2022-8-cores
     timeout-minutes: 30
     steps:
+    - name: Enable WSL2
+      run: |
+        wsl --set-default-version 2
+        # Manually install the latest kernel from MSI
+        Invoke-WebRequest -Uri "https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi" -OutFile "wsl_update_x64.msi"
+        $pwd = (pwd).Path
+        Start-Process msiexec.exe -Wait -ArgumentList "/I $pwd\wsl_update_x64.msi /quiet"
+        wsl --update
+        wsl --status
+        wsl --list --online
+    - name: Install WSL2 distro
+      timeout-minutes: 3
+      run: |
+        # FIXME: At least one distro has to be installed here,
+        # otherwise `wsl --list --verbose` (called from Lima) fails:
+        # https://github.com/lima-vm/lima/pull/1826#issuecomment-1729993334
+        # The distro image itself is not consumed by Lima.
+        # ------------------------------------------------------------------
```

Comment:
I’d like to see this resolved. Can be another PR though.

Reply:
I'll keep looking into this separately. Maybe there's something we can do with conditionally calling `wsl --import` / `wsl --install` based on the status returned from `wsl --list -v`